content/blog/breaks/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Breaks
+date: "2019-04-16T22:12:03.284Z"
+description: Replenish your energy.
+---
+
+> A break at work is a period of time during a shift in which an employee is allowed to take time off from his/her job. (Wikipedia).
+
+### Guilt
+
+Workers often feel guilty if they stand up and take a break. It's especially true for people in tech and the newcomers like interns or junior programmers. Hustling hard is the new norm, and I don't have anything against hard work, but play it smart to avoid getting [burned out](https://devhealth.io/burnout/).
+
+### Benefits
+
+Taking scheduled breaks during work is essential. It improves focus, keeps you from getting bored and helps you figure out a problem that you are stuck on.
+
+When you are frustrated and stressing out over a hard task, taking a break and putting the work in subconscious mind may help solve it.
+
+Walking, talking to coworkers or playing a video game can help you relax, and the insights will come more often when in a relaxed state.
```

Comment:
_often_

it is not wrong, but I would say 'easily' if the meaning is still okay